draft-ietf-rtcweb-jsep.xml

```diff
@@ -1437,7 +1445,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>If FEC is supported for this media type, another "a=ssrc"
               line with the FEC SSRC, and an "a=ssrc-group" line, as specified
               in <xref target="RFC5576"></xref>, section 4.2, with semantics
-              set to "FEC" and including the primary and FEC SSRCs.</t>
+              set to "FEC-FR" and including the primary and FEC SSRCs. For
```

Comment:
You need a reference to 5956 for FEC-FR. I have no opinion on whether we want FEC-FR or FEC, but I note that this is a change.